encounters.json

```diff
@@ -311,6 +311,9 @@
         "code": "shattered_aeons",
         "name": "Shattered Aeons"
     },
+    {
+        "code": "turn_back_time",
+        "name": "Turn Back Time"
```

Comment:
@drawntotheflame looks like this is missing a `},`

Reply:
Thanks @zzorba - price of getting distracted part way through